docs/reference/remap/expressions/function_call.cue

```diff
@@ -86,7 +86,7 @@ remap: expressions: function_call: {
 							If the type of the value is not known, you need to handle the potential argument error:
 
 							```vrl
-							number, err = round(.message)
+							number, err = round(int!(.message))
```

Comment:
`round` isn't fallible. So with the `int!` call ensuring it recieves an `int` it will never fail, so there is no `err` to handle.

You could just do:

```
number = round(int!(.message))
```

If  `.message` isn't an integer, the script will terminate with an error.

Or you could do:

```
number = round(int(.message) ?? 0)
```

Which would default to 0 if `.message` is not an integer.